UI/WCTDataTreeTabSample/WCTDataTreeTabSample/WCTDataTreeTabSample.Shared/MainPage.xaml

```diff
@@ -0,0 +1,14 @@
+﻿<Page
```

Comment:
This file should not be there, and it should use the Shell.xaml. Also, in Pages, make sure to restore all the other ones that have been removed.

Reply:
I did! It was my strategy to fix them one by one.

Reply:
The comment from @jeromelaban  is still applicable - MainPage.xaml and MainPage.xaml.cs should not be here as they're not used